### DISCUSSION TITLE
Prometheus Operator support for existing Prometheus set-up (not deployed through Prometheus operator)

### DISCUSSION THREAD

**Discussion by raj-gupta271** [prometheus-operator/kube-prometheus#1574] [Q&A] [answered]
Can proemtheus operator manage all the feature in Prometheus Server(deployed through deployement file notusing CRDs)? I mean if i deploy prometheus operator in existing prometheus set-up(kind: Deployment) and using the PrometheusRule CRD, can it update proemtheus.yaml config file and do needful change in rule file of prometheus.
I just want to know if i don't want to deploy new Prometheus Server, can we do only Alert Rule automation in Prometheus(kind:deployment not in kind:Prometheus) server. 
cc @hemant66wave

**Comment by paulfantom** [comment, marked as answer]
No, such a setup is not possible. The operator needs to manage prometheus server in full to allow all features to work correctly.
